### PR TITLE
Append external CA certs to certificateAuthority array only if they exist

### PR DESCRIPTION
**Pull request by timgerstel**

### CODE REVIEW

File: bin/convert-env.sh [hunk 1]
```diff
@@ -73,7 +73,12 @@ then
   if [ "$KEYSTORE_TYPE" = "JCERACFKS" ]
   then
     #, at end turns it into an array
-    export ZWED_node_https_certificateAuthorities="${TRUSTSTORE}&localca","${TRUSTSTORE}&${EXTERNAL_ROOT_CA}"
+    if [ -n "$EXTERNAL_ROOT_CA" ]
+    then
+      export ZWED_node_https_certificateAuthorities="${TRUSTSTORE}&localca","${TRUSTSTORE}&${EXTERNAL_ROOT_CA}"
+    else
+      export ZWED_node_https_certificateAuthorities="${TRUSTSTORE}&localca"
```
Comment (vit-tomica) [inline bin/convert-env.sh:80]: Shouldn't comma be here at the end?

Comment (timgerstel) [reply bin/convert-env.sh]: Yes you are correct

**Comment by vit-tomica** [comment]
Hello @timgerstel. It seems that this change has not been propagated to the zowe:staging branch yet. Can we get that to the staging branch, please? I'd like to create a new PR onto that. It's related to https://github.com/zowe/zowe-install-packaging/issues/1669 and https://github.com/zowe/zowe-install-packaging/pull/1746.

**Comment by struga0258** [comment]
Hi @vit-tomica, this should be fixed now it was merged into staging yesterday:

https://github.com/zowe/zlux-app-server/pull/141

**Comment by vit-tomica** [comment]
@struga0258 Thank you!

**Pull request merged**